### PR TITLE
chore: remove optimism Koran from configs and options etc

### PR DESCRIPTION
**Pull request by carsonfarmer**
Signed-off-by: Carson Farmer <carson.farmer@gmail.com>

## Summary

This PR removes all references and settings that pertain to optimism kovan, which has now been deprecated from tableland.

## Details

This is primarily changing options and settings. The core functionality of this library and contracts should be largely unchanged.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: contracts/utils/TablelandDeployments.sol [hunk 1]
```diff
@@ -60,8 +57,6 @@ library TablelandDeployments {
             return ITablelandTables(POLYGON);
         } else if (block.chainid == 5) {
             return ITablelandTables(GOERLI);
-        } else if (block.chainid == 69) {
```
Comment (carsonfarmer) [inline contracts/utils/TablelandDeployments.sol:63]: I'm less familiar with this contract code here @sanderpick. Does removing this (and thereby changing the compiled contract code) have any significant impact?

File: proxies.ts [hunk 2]
```diff
@@ -9,12 +9,10 @@ export const proxies: ProxyAddresses = {
   polygon: "0x5c4e6A9e5C1e1BF445A062006faF19EA6c49aFeA",
   // tableland testnet testnets
   "ethereum-goerli": "0xDA8EA22d092307874f30A1F277D1388dca0BA97a",
-  "optimism-kovan": "0xf2C9Fc73884A9c6e6Db58778176Ab67989139D06",
```
Comment (carsonfarmer) [inline proxies.ts:12]: Removing this here also removes it from the compiled `proxies.js`. This means downstream consumers of proxies.js (namely the SDK) will need a new version to pull the updated proxies from.

**Review by joewagner — APPROVED** [review]
I don't know the answer to the question you asked Sander off the top of my head, but this all looks good to me

File: contracts/utils/TablelandDeployments.sol (hunk 1 above)
Comment (sanderpick) [reply contracts/utils/TablelandDeployments.sol:63]: nope, just means people won't be able to use the deployments lib to get the kovan address. nbd.

**Review by sanderpick — APPROVED** [review]
LGMT!

**Pull request merged**